cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -271,13 +272,28 @@ func TestPolicyUpdate(t *testing.T) {
 			},
 		},
 		"Other user, update old data": {
-			args:       []string{"UserIntegrationTest@example.com", "UserIntegrationTest@example.com.krb5"},
+			args:       []string{"userintegrationtest@example.com", "userintegrationtest@example.com.krb5"},
```

Comment:
Oh, I thought was could keep the .kbr5 file with the current case so that we distinguish between the username and the filename.
However, when we add the symlink, the symlink is lowercase as this is what adsys would create. Am I wrong?